database/migrations/legacy/2022_12_06_000000_adjust_contrib_badges.php

```diff
@@ -0,0 +1,108 @@
+<?php
```

Comment:
Migrations are usually meant to be "just" schema migrations - personally I'd implement migrations for existing data like calculation or moving data around as commands. 

Comparable to the sync commands, this would keep migrations fast (think running migrations during testing where this data migration is not relevant) and lets more time intensive tasks to be run on-demand and repeatedly.

There are a few commands in the `ra:` namespace already: `php artisan list | grep "ra:"`

Could be something like `ra:server:developer:recalc-contrib-yield` in this case?